.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: scala
+scala: 2.12.7
+jdk: openjdk8
+
+before_install:
+  - sudo apt-get -y install nodejs
+
+cache:
+  directories:
+    - $HOME/.sbt
+    - $HOME/.ivy2
+
+before_cache:
+  # Cleanup the cached directories to avoid unnecessary cache updates
+  - rm -fv $HOME/.ivy2/.sbt.ivy.lock
+  - find $HOME/.ivy2/cache -name "ivydata-*.properties" -print -delete
+  - find $HOME/.sbt        -name "*.lock"               -print -delete
+
+script:
+  - sbt checkAllFormat
+  - sbt clean coverage test
```

Comment:
We have to run `crossJS/test` and `crossJVM/test` separately to avoid testing on the root project. This will be a little bit annoying, though. Since all the code are the same for both platform, maybe we can run the test only on JVM (using the [withoutSuffixFor](https://github.com/portable-scala/sbt-crossproject#removing-the-platform-suffix-for-one-default-platform) setting).

Reply:
Got it. Thanks for pointing out. I will give it a shot.